torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -1367,7 +1368,7 @@ def _materialize_module(
         ``reset_parameters()``, and for torchdistX fake tensors, this calls
         ``deferred_init.materialize_module()``.
         """
-        is_meta_module = any(p.is_meta for p in module.parameters())
+        is_meta_module = any(p.is_meta for p in self._get_orig_params(module, ignored_params))
```

Comment:
nit: the fake.is_fake check on 1375 may also have to be updated. But feel free to file an issue if not in scope of this PR.